requirements.txt

```diff
@@ -9,3 +9,4 @@ django-environ==0.11.2
 uuid==1.30
 djangorestframework-simplejwt==5.3.1
 django-cors-headers==4.6.0
+factory_boy==3.3.1
```

Comment:
this should go in requirements-dev.txt, since it will not be used in prod

Reply:
Makes sense. I'll move it.

Reply:
Done